src/test/java/com/serenitydojo/WhenWorkingWithWholeNumbers.java

```diff
@@ -0,0 +1,21 @@
+package com.serenitydojo;
+
+
+import org.junit.Test;
+
+
+public class WhenWorkingWithWholeNumbers {
+
+    @Test
+    public void addNumbersTogether(){
+        int initialYear =  1985;
+        int targetYear  = 0;
```

Comment:
same as https://github.com/serenity-dojo/exercise-1-kingsedem/pull/1/files#r1069047911

Reply:
Done

Reply:
not pushed yet?